### PR TITLE
[runtime] Test GetPid for Mono.Posix

### PR DESCRIPTION
**Pull request by alexanderkyte**

### CODE REVIEW

File: mcs/class/Mono.Posix/Test/Mono.Unix.Native/StdlibTest.cs [hunk 1]
```diff
@@ -29,6 +29,15 @@ public void Handler (int sn)
 			}
 		}
 
+
+		[Test]
+		public void GetPid ()
+		{
+			var currentPID = Mono.Unix.Native.Syscall.getpid();
+			Console.WriteLine ("CurrentPID is {0}", currentPID);
```
Comment (akoeplinger) [inline mcs/class/Mono.Posix/Test/Mono.Unix.Native/StdlibTest.cs]: please remove this unnecessary logging, we'll get the value in the assertion message anyway.

File: mcs/class/Mono.Posix/Test/Mono.Unix.Native/StdlibTest.cs [hunk 2]
```diff
@@ -29,6 +29,15 @@ public void Handler (int sn)
...
+			Assert.AreNotEqual (currentPID, 0);
```
Comment (akoeplinger) [inline mcs/class/Mono.Posix/Test/Mono.Unix.Native/StdlibTest.cs]: the arguments need to be switched, it's `(expected, actual)` not the other way round :)

might make sense to use `Assert.Greater(currentPID, 0)` so we verify we don't get a negative value too.

Comment (jaykrell) [reply mcs/class/Mono.Posix/Test/Mono.Unix.Native/StdlibTest.cs]: Who says negative values are invalid?
http://pubs.opengroup.org/onlinepubs/007904875/functions/getpid.html ? no.
mac man posix? no.

Comment (jaykrell) [reply mcs/class/Mono.Posix/Test/Mono.Unix.Native/StdlibTest.cs]: 0 is valid too. There is no reserved value.

Comment (akoeplinger) [reply mcs/class/Mono.Posix/Test/Mono.Unix.Native/StdlibTest.cs]: Right but the fact that every OS we care about returns positive PIDs and uses PID0 for scheduling/system processes makes me confident in this being fine for testing code :)

**Comment by luhenry** [comment]
@alexanderkyte what's the progress on this PR?

**Comment by luhenry** [comment]
@monojenkins build

**Pull request merged**